app/controllers/reports_controller.rb

```diff
@@ -20,6 +20,7 @@ def history
       else
         @records = GameHistoryDao::get_per_game_histories(primary_player_id)
       end
+      @records = @records.sort_by{|k,v| k}.reverse
```

Comment:
When you say `unbound`, are you thinking that `sort_by` could potentially be called on a null object?